tools/packages/packageSchema.yaml

```diff
@@ -56,3 +56,52 @@ stages:
       - publish: $(pkgOut)
         artifact: bis-schemas-packages
         displayName: 'Publish Pipeline Artifact'
+
+- stage: Generate
+  dependsOn: 
+  - Build
+  condition: succeeded('Build')
+  jobs:
+  - job: generate_jsons
+    displayName: Generate JSONS
+    pool:
+      vmImage: windows-latest
+
+    workspace:
+      clean: all
+    variables:
+    - name: jsonOut
+      value: '$(Agent.BuildDirectory)/JSONS'
+    
+    steps:
+      - task: NodeTool@0
+        displayName: 'Use Node 14.16.0'
+        inputs:
+          versionSpec: 14.16.0
+      - checkout: self
+        clean: true
+      
+      - powershell: |
+          $npmrcDir = "$(Build.Repository.LocalPath)"
+          
+          if(!(Test-Path -Path $npmrcDir)){
+              mkdir $npmrcDir
+          }
+          
+          $npmrcFile = $npmrcDir + "/.npmrc"
+          
+          Out-File -FilePath $npmrcFile -Encoding ascii -InputObject "@bentley:registry=https://registry.npmjs.org/"

```

Comment:
Do we need this?  We're just pulling from public npm

Reply:
removed the need for seeding npm rc in previous commit